torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -561,6 +561,14 @@ TORCH_CUDA_CU_API TensorView* gather(
     const std::vector<int>& strides = {},
     bool trim_out_of_bounds = false);
 
+//! A fused multiply and add operator that will
```

Comment:
To me, this name sounds a little confusing as "mma" stands for matrix multiply accumulate, but there's no accumulate in this expression. An mma operation is internally used, but that's an implementation detail.

Reply:
This op does accumulate into the output register, similar the the reductionOp. The comment said FMA since that's kind of what mma instruction really is but I'm not sure if we'd use it in FMA mode anytime soon.

On the side not sure what "accumulate" in mma refers to. My guess is it's accumulating in the small matmul macro. But the mma instruction semantic is really a "matrix FMA", but I currently only used it as an accumulator.

Reply:
I don't agree. Accumulate in the context of these matrix operations should mean `D= X * A * B+ C`.  See Figure 5 of this page: https://developer.nvidia.com/blog/programming-tensor-cores-cuda-9/. We don't have `C` here.

This is a user function, so the function name should reflect what the user-level semantics is.

Reply:
Sure. This looks like name collision I will just rename this one to `matmul`.